src/components/Modal/index.js

```diff
@@ -1,139 +1,72 @@
-import React, { Component } from 'react'
-import { bool, func, node, string, oneOfType, arrayOf, oneOf } from 'prop-types'
+import React, { PureComponent } from 'react'
+import { createPortal } from 'react-dom'
+import PropTypes from 'prop-types'
 import cn from 'classnames'
 
-import ModalPortal from '../ModalPortal'
-import ModalHeader from '../ModalHeader'
-import ClickOutside from '../ClickOutside'
-import RoundedBox from '../RoundedBox'
+import IconClose from '../Icons/Close'
 
-import Close from '../Icons/Close'
-
-export default class Modal extends Component {
-  static CLOSE_POSITIONS = {
-    INSIDE: 'inside-right',
-    INSIDE_RIGHT: 'inside-right',
-    INSIDE_LEFT: 'inside-left',
-    OUTSIDE: 'outside-right',
-    OUTSIDE_RIGHT: 'outside-right',
-    OUTSIDE_LEFT: 'outside-left',
-    NONE: 'none',
-  }
 
+export default class Modal extends PureComponent {
   static propTypes = {
-    isOpen: bool.isRequired,
-    onClose: func.isRequired,
-    children: oneOfType([arrayOf(node), node]).isRequired,
-    shouldCloseOnEsc: bool,
-    shouldCloseOnClickOutside: bool,
```

Comment:
Are we sure that we don't want to close the modal on click outside?

Reply:
Removing this property is not removing that functionality, it still remains.